api/src/main/resources/config/application-dev.properties

```diff
@@ -5,6 +5,7 @@ app.baseUrl=http://localhost:8080
 
 # Spring Boot will serve frontend files via port 8080
 # If you are using ng serve to proxy static files built by Angular, set OSMT_FRONT_END_PORT to 4200
+#app.frontEndPort=${OSMT_FRONT_END_PORT:4200}
```

Comment:
There's no need to change this. If you want to change the value, provide an env var called "OSMT_FRONT_END_PORT".